assets/src/edit-story/components/panels/document/excerpt/excerpt.js

```diff
@@ -66,20 +73,21 @@ function ExcerptPanel() {
           ref={ref}
           value={excerpt}
           onChange={handleTextChange}
-          placeholder={__('Write a description of the story', 'web-stories')}
-          aria-label={__('Story Description', 'web-stories')}
+          placeholder={__('Write an excerpt', 'web-stories')}
+          aria-label={__('Excerpt', 'web-stories')}
           maxLength={EXCERPT_MAX_LENGTH}
+          showCount
           rows={4}
           css={highlight?.showEffect && styles.OUTLINE}
         />
       </Row>
       <Row>
-        <Note>
+        <StyledText size={THEME_CONSTANTS.TYPOGRAPHY.PRESET_SIZES.X_SMALL}>
```

Comment:
Did you figure out the issue with this not being applied if added directly to `styled(Text).attrs({ size: ... })`?

Reply:
Yep, Samuel had [thoughts](https://github.com/google/web-stories-wp/pull/6784#discussion_r596153103) about this -- apparently `forwardedAs: 'span'` needed using instead of just `as`.